tools/swift-reflection-dump/swift-reflection-dump.cpp

```diff
@@ -107,10 +115,19 @@ static bool needToRelocate(SectionRef S) {
 
   if (auto EO = dyn_cast<ELFObjectFileBase>(S.getObject())) {
     static const llvm::StringSet<> ELFSectionsList = {
-      ".data", ".rodata", "swift5_protocols", "swift5_protocol_conformances",
-      "swift5_typeref", "swift5_reflstr", "swift5_assocty", "swift5_replace",
-      "swift5_type_metadata", "swift5_fieldmd", "swift5_capture", "swift5_builtin"
-    };
+        ".data",
+        ".rodata",
+        ".note.swift_reflection_metadata",
+        "swift5_protocols",
+        "swift5_protocol_conformances",
+        "swift5_typeref",
+        "swift5_reflstr",
+        "swift5_assocty",
+        "swift5_replace",
+        "swift5_type_metadata",
+        "swift5_fieldmd",
+        "swift5_capture",
+        "swift5_builtin"};
```

Comment:
Please add a `,` at the end of the entry here so that `clang-format` doesn't reformat this.

Reply:
done